server/nativeapi/playlists.go

```diff
@@ -42,6 +44,35 @@ func getPlaylist(ds model.DataStore) http.HandlerFunc {
 	}
 }
 
+func createPlaylistFromM3U(ds model.DataStore) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		ctx := r.Context()
+		user, ok := request.UserFrom(ctx)
+		if !ok {
+			log.Error(ctx, "Unable to retrieve user info from request contest")
+			w.WriteHeader(http.StatusInternalServerError)
+			return
+		}
+
+		p := core.NewPlaylists(ds)
```

Comment:
We already have the `core.Playlists` instantiated in the Router, we could pass it as a parameter, similar to what we are doing with the `model.DataStore`.